pyroapi/tests.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from pyroapi.dispatch import distributions as dist
+from pyroapi.dispatch import infer, ops, optim, pyro
+
+
+@pytest.mark.parametrize("jit", [False, True], ids=["py", "jit"])
+@pytest.mark.parametrize("optim_name", ["Adam", "ClippedAdam"])
+def test_optimizer(backend, optim_name, jit):
```

Comment:
Note this backend arg must be provided by a user-defined fixture, as in the tests.